_posts/2016/2016-03-16-team-orange-mallorca.md

```diff
@@ -0,0 +1,41 @@
+
+{% image blog/blog-mallorca-3.jpg %}{% endimage %}
+
+There is always time for everything and so we had enough time to work on our technological and business challenges, too. We accomplished two knowledge sharing sessions, one by our pipeline expert in order to understand how it works, another one on language handling and language packs.
+
+{% image blog/blog-mallorca-4.jpg %}{% endimage %}
+
+We also played two rounds of [Coding Dojo](http://codingdojo.org), where we focused on solving the Kata of Conway’s Game of Life. In the first round we tried to apply the constraints “TDD as if you meant it” and “Driver & Navigator“, which strictly seperates the roles in pair programming. The second round focused on shaping distinct, comprehensive and sustainable test cases. Therefore, one ePagee had to implement an unit test. Next, the coding partner should lean into the position of an evil programmer and hence only implement as little as possible to make the test pass. Afterwards he gives back the keyboard to the radically stressed writer of the unit test, who has to enforce the desired implementation with the addition of another test case. In this rotary principle we trained our sense for writing high quality tests. This hands-on approach delivered a lot of amusements as well as practical benefits for our programming skills.
+
+{% image blog/blog-mallorca-5.jpg %}{% endimage %}
+
+At one day a hackathon accelerated our ingenious chains of thoughts. From the early afternoon until late into the night three pairs took the opportunity to delve into topics of their interest. One pair evaluated the Jenkins [Pipeline Plugin](https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Plugin) and built a prototype with multiple continuous delivery stages. Others automated the handling around our translation process. The third pair developed a new concept how to deal with failing tests – of course not without a working prototype. Presenting the results to the other pairs the next morning showed how quick ideas can grow into applicable solutions.
+
+Two [Design Thinking](http://hpi.de/school-of-design-thinking.html) sessions took us beyond the mental barrier of longsome beaten paths. The consistent execution of this method over and over again rapidly helped us to find the wanted ideas for our current team mission. Firstly, we want to return a lot of responsibilities to those teams that strongly depend on us in these days. Secondly, we want to establish a fast and secure mechanism to permanently keep the latest build of the next ePages version in a releasable state.
```

Comment:
...the wanted ideas... >... the desired ideas...

...depend on us in these days. > depend on us these days.

Secondly, we want to establish a > Secondly, we'd like to establish a...

latest build of the next ePages version > does that refer to ePages 6 software? or is  that an "outlook"?